src/modules/cmd_duration.rs

```diff
@@ -45,14 +45,16 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 }
 
 // Render the time into a nice human-readable string
-fn render_time(raw_seconds: u64) -> String {
+fn render_time(raw_nanoseconds: u64) -> String {
     // Calculate a simple breakdown into days/hours/minutes/seconds
+    let raw_milliseconds = raw_nanoseconds / 1_000_000;
```

Comment:
Good usage of numeric separators--I think I would have gone slightly insane without them. :+1:

Reply:
Thanks! I am rather new to rust. I didn't actually know these existed until I ran the linter on the code. It's a great feature though!